### DISCUSSION TITLE
Getting error everytime I open a terminal/start up the prompt

### DISCUSSION THREAD

**Discussion by giomxrcode** [JanDeDobbeleer/oh-my-posh#5612] [Troubleshoot] [answered]
Hello! I've recently installed OMP and been getting an error everytime I open up the terminal or start the shell with OMP. First time I'm getting this error, cause it worked ok previously for me. It only happens when the prompt/shell starts, but works normally after typing commands and stuff (unless I get an error exit code).

I haven't found and answer, just a discussion from a while ago, but the only solutions seems to be to install a previous version of OMP. 

![image](https://github.com/user-attachments/assets/8ab1ddf2-3284-4227-a11b-d0f45523286d)
This is a custom prompt theme

![image](https://github.com/user-attachments/assets/df64caf6-9f88-4af6-86db-f30d27f06f66)
This is the default prompt theme

**Comment by JanDeDobbeleer** [comment, marked as answer]
@giomxrcode this means that something is your shell startup exits with a non zero exit code. Oh-my-posh only shows what the shell informs, it's not an issue in oh-my-posh but something else that's having an issue.

**Reply by giomxrcode** [reply]
Yeah, it was! I couldn't find what the issue was, but I rewrote my .zshrc and reinstalled some of my plugins, and it worked perfectly. Thanks for the tip!
